gpytorch/kernels/grid_kernel.py

```diff
@@ -133,7 +134,7 @@ def forward(self, x1, x2, diag=False, last_dim_is_batch=False, **params):
                 ]  # Each entry i contains a grid_size[i] x grid_size[i] covariance matrix
                 if last_dim_is_batch:
                     # Note that this requires all the dimensions to have the same number of grid points
-                    covar = gpytorch.cat([c.unsqueeze(-3) for c in covars], dim=-3)
+                    covar = gpytorch_cat([c.unsqueeze(-3) for c in covars], dim=-3)
```

Comment:
Any reason for the `gpytorch.cat -> gpytorch_cat` change?

Reply:
Couple of reasons:
1. I think it's good to be consistent with relative vs absolute imports and seems like in the vast majority of cases you do relative imports
2. I imported it as `gpytorch_cat` instead of just `cat` to retain the clarity that it is the `gpytorch` version of cat rather than `torch.cat`...though in retrospect, just importing as `cat` would maybe be preferable